src/agent/AgentLogger.js

```diff
@@ -38,26 +39,42 @@ function create(agentId) {
         };
     }
 
-    let stream = createCWStream({
+    let cloudWatchtream = createCWStream({
         logGroupName: process.env.GTM_AGENT_CLOUDWATCH_LOGS_GROUP || 'gtmAgent',
         logStreamName: 'AGENT_ID=' + agentId,
         cloudWatchLogsOptions: CWLogOptions
-    });
+    }).on('error', console.error);
 
     janitorInterval = setInterval(streamJanitor, 60000);
 
-    return bunyan.createLogger({
+    let bunyanConf = {
         name: agentId.substring(0, 7),
         streams: [
             {
-                stream: stream,
+                stream: cloudWatchtream,
```

Comment:
Is this and the initial creation meant to be 'tream' or 'stream' :)

Reply:
ah yes.  standby

Reply:
fixed